src/System.Net.Sockets/src/System/Net/Sockets/AcceptOverlappedAsyncResult.Windows.cs

```diff
@@ -58,7 +58,7 @@ internal override object PostCompletion(int numBytes)
                         SocketOptionLevel.Socket,
                         SocketOptionName.UpdateAcceptContext,
                         ref handle,
-                        Marshal.SizeOf(handle));
+                        IntPtr.Size);
```

Comment:
`sizeof(IntPtr)` only works in unsafe context. There is nothing unsafe about it - there are several issues in Roslyn 

So I have used IntPtr.Size where possible so that I do not have to sprinkle the unsafe contexts around.